File: src/main/java/br/anhembi/funmodechild/model/common/Address.java
package br.anhembi.funmodechild.model.common;

import lombok.Data;

@Data
public class Address {
    private String addressLine;
    private String number;
    private String addressComplement;
    private String neighbourhood;
    private String postalCode;
    private String city;
    private String state;
}

File: src/main/kotlin/br/anhembi/funmodechild/model/common/Address.kt
package br.anhembi.funmodechild.model.common

data class Address(
    val addressLine: String? = null,
    val number: String? = null,
    val addressComplement: String? = null,
    val neighbourhood: String? = null,
    val postalCode: String? = null,
    val city: String? = null,
    val state: String? = null
)
